test/dune

```diff
@@ -1,4 +1,3 @@
-(test
- (name test_circle)
- (modules test_circle)
+(executable
+ (name test_all)
```

Comment:
Ideally, I would have wanted single executables for each, and then one command to run them all.
I tried the `progn` stanza but again ran into the same problem. This looks simplest for the moment.